runtime/ftplugin/query.lua

```diff
@@ -2,5 +2,21 @@
 -- Language:	Tree-sitter query
 -- Last Change:	2022 Mar 29
 
+if vim.b.did_ftplugin == 1 then
+  return
+end
+
+if vim.g.query_disable_linter ~= 1 then
+  local buf = vim.api.nvim_get_current_buf()
+  vim.api.nvim_create_autocmd(vim.g.query_linter_events or { 'BufWrite', 'CursorHold' }, {
+    group = vim.api.nvim_create_augroup('querylint', {}),
+    buffer = buf,
+    callback = function()
+      require('vim.treesitter.query_linter').lint(buf)
```

Comment:
Personally I would reserve submodules for commonly used stuff. I think playground and query_linter fall outside of that.

Reply:
Agreed, I don't think this needs to be exposed. (This was an "_if_" comment...)

Personally, I'd put the query linter into the `playground` module as well. (Which I think would be better named `devtools`, but that's just me.)

Reply:
`devtools` would be great. I just created a separate module in order to have to tied functions: lint/clear. Otherwise, the clear could be dropped or prefixed: query_linter_lint/query_linter_clear

Reply:
`playground` is now fixed as the name, and everything "playground" related should be in there. You can follow the `vim.treesitter.inspect_tree()` approach for exposing it.

`lint_query()` and `lint_clear()` would be fine with me.

Reply:
Followed the `playground` approach. Except that I made `query_linter.lua` have leading underscope (`_query_linter.lua`). Is it needed to expose the file visibly to users. It might be moved to devtools in future.